JSONResourceFile.js

```diff
@@ -234,27 +237,29 @@ JSONResourceFile.prototype.getContent = function() {
  */
 JSONResourceFile.prototype._calcLocalePath = function(locale, type, filename) {
     var rootLocale = "en-US";
-    var loc = new Locale(locale);
-    var lo = loc.getSpec();
-    var path = "";
-    if (!type) type = "js";
-    var resDir = this.project.getResourceDirs("json")[0] || ".";
-    var mappingData = this.getMapping(type);
+    var lo = new Locale(locale);
+    var fullPath = "";
 
     if (this.baseLocale) {
-        if (locale !== rootLocale) {
-            lo = loc.getLanguage();
+        if (this.locale.getSpec() !== rootLocale) {
+            fullPath = "/" + lo.getLanguage();
         }
+    } else {
+        var nodeVersion = process.versions["node"].split(".")[0];
+        if (nodeVersion < 15) {
+            fullPath += "/" + locale.getSpec().replace(/-/g, "/");
```

Comment:
You can use this line on all versions of node

Reply:
I updated it